### PR TITLE
Support += operator for SQLQueryString.

### PR DESCRIPTION
**Pull request by maciejtrybilo**
Very simple change to support the += operator for appending SQLQueryStrings.

### CODE REVIEW

**Review by 0xTim — APPROVED** [review]
LGTM - any objections @gwynne ?

File: Sources/SQLKit/Query/SQLQueryString.swift [hunk 1]
```diff
@@ -119,6 +119,10 @@ extension SQLQueryString {
     public static func +(lhs: SQLQueryString, rhs: SQLQueryString) -> SQLQueryString {
         return "\(lhs)\(rhs)"
     }
+    
+    public static func +=(lhs: inout SQLQueryString, rhs: SQLQueryString) {
+        lhs = "\(lhs)\(rhs)"
```
Comment (gwynne) [inline Sources/SQLKit/Query/SQLQueryString.swift:124]:
```suggestion
        lhs.fragments.insert(rhs,.fragments, at: lhs.fragments.startIndex)
```
This avoids the creation of an unnecessary intermediate object.

**Review by gwynne — CHANGES_REQUESTED** [review]
One nit

File: Sources/SQLKit/Query/SQLQueryString.swift (hunk 1 above)
Comment (maciejtrybilo) [reply Sources/SQLKit/Query/SQLQueryString.swift]: Thanks!

**Review by gwynne — APPROVED** [review]
LGTM, just waiting for CI to pass now

File: Sources/SQLKit/Query/SQLQueryString.swift [hunk 2]
```diff
@@ -119,6 +119,10 @@ extension SQLQueryString {
     public static func +(lhs: SQLQueryString, rhs: SQLQueryString) -> SQLQueryString {
         return "\(lhs)\(rhs)"
     }
+    
+    public static func +=(lhs: inout SQLQueryString, rhs: SQLQueryString) {
+        lhs.fragments.insert(contentsOf: rhs.fragments, at: lhs.fragments.startIndex)
```
Comment (gwynne) [inline Sources/SQLKit/Query/SQLQueryString.swift:124]: Whoops, my mistake - I got it a little backwards 🤦‍♀️ It should be:
```suggestion
        lhs.fragments.append(contentsOf: rhs.fragments)
```

**Review by gwynne — CHANGES_REQUESTED** [review]
Sorry about that! I got the argument order backwards in my suggested code 😅.

**Review by gwynne — APPROVED** [review]
Okay, let's try that one more time! 😆

File: Sources/SQLKit/Query/SQLQueryString.swift (hunk 2 above)
Comment (maciejtrybilo) [reply Sources/SQLKit/Query/SQLQueryString.swift]: Oops, I should have checked that :)

**Pull request merged**